### PR TITLE
Development

### PR DESCRIPTION
**Pull request by suretrust**
Submitting for code review.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
+body {
+    font-family: 'Open Sans', Arial, sans-serif;
```
Comment (inhaq) [inline css/style.css]: Good work on adding `font-family` but you'll need to add `link` in HTML as well for the font. Check out [Google fonts](https://fonts.google.com/specimen/Open+Sans).

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+#signup{
+    background-color: #ffffff;
+    border: 1px solid #c7c7c7;
+    color: #4C4C4C;
+    max-width: 378px;
+    margin: 0 auto;
+}
+
+#signup section {
```
Comment (inhaq) [inline css/style.css]: Consider looking into [this resource](https://css-tricks.com/the-difference-between-id-and-class/) in order to learn more about CSS class and id. Kindly figure out what you are doing wrong here.
Hint: Problem is with selectors on line 7 and 15.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    background-image: url("../img/unlocked.png");
+    background-repeat: no-repeat;
+    background-position: 98% 50%;
+}
+
+#call{
+    font-size: 10px;
+}
+
+button {
```
Comment (inhaq) [inline css/style.css:54]: Instead of using HTML tags directly, I would recommend using classes.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Mint Sign In Page</title>
+</head>
+<body>
+    <form id="signup" action="#">
+
+        <section>
+                <center>
```
Comment (inhaq) [inline index.html]: The `center` element is obsolete. Use CSS instead.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+                    </fieldset>
+            
+                    <fieldset>            
+                        <label for="password">Password</label>
+                        <input type="password" id="password" name="password">
+                    </fieldset>
+                        
+                    <button type="submit">Create Account</button><br /><br /><br />
+            
+                    <center>
```
Comment (inhaq) [inline index.html]: The `center` element is obsolete. Use CSS instead.

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+    font-size: 16px;
+    border-radius: 5px;
+    background-image: url("../img/locked.png"), linear-gradient(to bottom,#167CCC 0,#0077C5 100%);
```
Comment (inhaq) [inline css/style.css:57]: Use `./img/locked.png` instead. Remove first dot as it means to go to the previous directory.

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+    width: 100%;
+    box-sizing: border-box;
+    border: 1px solid #C1C5C8;
+    border-radius: 3px;
+    line-height: 1.3em;
+    padding: 8px;
+}
+
+input[type="password"] {
+    background-image: url("../img/unlocked.png");
```
Comment (inhaq) [inline css/style.css:45]: Use `./img/unlocked.png` instead. Remove first dot as it means to go to the previous directory.

**Review by inhaq — CHANGES_REQUESTED** [review]
### Great work on this project 🏆
Kindly look into these changes mentioned below. 
You can improve this project by adding the logo of Intuit as well.
In order to see whether your HTML is good, you can validate it [here](https://validator.w3.org). Once you finished with the changes and improvements, kindly resubmit it and mentioned me(Inzimam) in reviewed before. Happy coding!

File: css/style.css (hunk 6 above)
Comment (suretrust) [reply css/style.css:57]: We pointed it there because that's where the file we need is.

**Comment by inhaq** [comment]
### Congrats 🎉 on completing this project
Feel free to submit a project completion form for this project, it's 👍🏻.
Suggestion: Don't use ID in CSS as selectors in the next projects. Happy Coding 😊

**Pull request merged**